src/backend/marsha/core/permissions.py

```diff
@@ -6,9 +6,10 @@
 
 from rest_framework import permissions
 
+from marsha.core.simple_jwt.authentication import TokenResource
```

Comment:
A reason to make this change ?

Reply:
Nope, I missed a change bellow for the `HasPlaylistToken`, need to fix it (don't use the `isinstance`, use the `request.resource` instead).